releases-beta/how-to-guides/resolving-a-cherry-pick-failure.md

```diff
@@ -0,0 +1,41 @@
+# Resolving a Cherry-Pick Failure
```

Comment:
Both for this file and Custom Workflow parameters, please add the links in `SUMMARY.md`. Without it these files do not appear in the docs nav.